tsl/src/compression/compression.h

```diff
@@ -11,7 +11,13 @@
 #include <executor/tuptable.h>
 #include <fmgr.h>
 #include <lib/stringinfo.h>
+#include <access/heapam.h>
 #include <utils/relcache.h>
+#include <utils/tuplesort.h>
```

Comment:
Do you need these 2 header files to be included ?
Adding unwanted header files will increase dependency between different c files and can cause increase in source code compilation times.

Reply:
<access/heapam> is needed for BulkInsertState bistate in RowCompressor

Reply:
Ok.